samples/nrf9160/modem_shell/src/gnss/gnss_socket.c

```diff
@@ -702,6 +688,36 @@ int gnss_stop(void)
 	return ret;
 }
 
+int gnss_delete_data(enum gnss_data_delete data)
+{
+	int ret;
+	nrf_gnss_delete_mask_t delete_mask;
+
+	gnss_init();
+
+	switch (data) {
+	case GNSS_DATA_DELETE_EPHEMERIDES:
+		delete_mask = 0x01;
```

Comment:
I think that using `NRF_MODEM_GNSS_DELETE_*` flags as in `gnss_interface.c` is more readable(although longer).

Reply:
I agree, but I didn't want to mix different GNSS APIs. gnss_socket.c uses the (now deprecated) GNSS socket API (nrf_socket.h), which doesn't have defines for the bit values. We added those in the new API (nrf_modem_gnss.h), but I don't want to include that in gnss_socket.h.

Reply:
Ah, ok, I didn't realize that. Nevermind then.